data/transition-sites/mod_sabre.yml

```diff
@@ -0,0 +1,6 @@
+---
+site: mod_sabre
+whitehall_slug: ministry-of-defence
+homepage: https://www.gov.uk/government/organisations/ministry-of-defence
```

Comment:
OK, let's leave the homepages as they are for now, and update them later. We'll also need to add `homepage_title` to describe the new `homepage` URLs when they change.